komodo/show_version.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+import argparse
+import yaml
+import os
+import re
+from pathlib import Path
+import configparser
+from typing import Union
+from typing import Optional
+
+
+def guess_release() -> str:
+    """
+    Try to sense the Komodo release from the active environment.
+    """
+    try:
+        return os.environ["KOMODO_RELEASE"]
+    except KeyError:
+        raise Exception("Could not sense the name of the release, please provide it.")
```

Comment:
We should not use bare exceptions. You may use a dummy exception `KomodoShowVersionException` that just inherits from `Exception` if needed.

Reply:
Changed to `RuntimeError`, what do you think? Definitely an improvement, not sure what's best.